docs/UserGuide.md

```diff
@@ -251,7 +251,89 @@ Examples:
 * `deletetask 2` deletes the task with index 2 from the task list.
 
 
+### Creating a Consult: `createconsult`
+Creates a consultation into upcoming consultations.
```

Comment:
'Creates a consultation into upcoming consultations.'
This sounds confusing, might want to rephrase it to:
Creates a new consultation.

Becareful of the naming as well. Line 254, you used 'createconsult', 
those below (i.e Line 265, 266)you used 'addconsult'